keanu-project/build.gradle

```diff
@@ -38,6 +38,7 @@ dependencies {
     //testing
     testCompile 'junit:junit:4.12'
     testCompile 'org.mockito:mockito-core:2.19.1'
+    testCompile 'org.assertj:assertj-core:3.11.1'
```

Comment:
We’ve been using Hamcrest, which is a slightly older assertion lib with similar functionality. They have a very similar api most the time but assertj does a nicer api for more complex assertions. I’ve used both libs before and I could go either way with it if we didnt already have quite a few tests that use Hamcrest. You could make an argument to have both but for simplicity sake I’d say just go with one.

Reply:
sounds good, switching to hamcrest